custom_components/powercalc/data/signify/LCL003/model.json

```
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly, Plug @\u007fS",
```

You have some encoding issue here.

```suggestion
    "measure_device": "Shelly Plug S",
```